js/src/collapse.js

```diff
@@ -162,7 +162,7 @@ const Collapse = (($) => {
 
       this._element.style[dimension] = 0
 
-      if (this._triggerArray.length > 0) {
+      if (this._triggerArray.length) {
```

Comment:
I'd keep the length checks

Reply:
That's useless because 0 is a false value

Reply:
Yep but that rule isn't activated

Reply:
I never said it was :P I remember I made the length changes some time ago, so I'd say keep them even though we don't have that check.

Reply:
I prefer to remove them, it's part of the JavaScript power 💪

Reply:
I don't agree, because I'd rather be explicit when possible. It's actually one of the problems of JavaScript, not strict types etc.

Reply:
That's a well known property of JavaScript that 0 is a false value, be explicit isn't a good thing when it's hide a language particularity because other languages are different

Reply:
Sorry, I don't agree with the length removal. Even if we know that, I always prefer to be explicit.

The rest of the patch looks good, though. 👍